src/components/ObjectForm.vue

```diff
@@ -346,14 +352,18 @@ export default {
             const filed = []
             const fieldDict = {}
 
-            for (const section of layout.editLayoutSections) {
+            for (const section of sections) {
               let fieldCount = 0
 
               for (const row of section.layoutRows) {
                 for (const row of section.layoutRows) {
```

Comment:
Is the double `for` with same items and shadowing intentional here?

Reply:
yep... it's complicated